### ISSUE TITLE
Missing backstory?

### ISSUE THREAD

**Issue by Stephen-Gates** [opendataboardgame/game#41]
The event cards have very detailed descriptions. It seems like there's a missing backstory...

Who are all these people: 
- Sheridan Mayor, Bob Brump
- Mayoral candidate, Les Lessig
- Landpoint Investments / Insights CEO, Natalie Pemberton-Smith
- Head of Sheridan Parks & Wildlife, Lisa Truss 
- 87 year old, Jack Jerry 

What are these organisations:
- Landpoint Investments, Fish farmers and land developers. Owners of Schumpeter Shopping Mall
- Dodds Cooperative, Sheridan Family grocer
- Pikesley Farms, the largest asparagus producer in the country. Has a factory and stores.  
- Ponting Premium Growers, Coalville-Based asparagus producer

Where are these places: 
- Sheridan, Pop: 1.2m, Declining town as shops close, teachers quit, hedgehogs go extinct and pollution rises
- Colville, neighbouring town and rival asparagus producer 
- Well's Wood, home of the endangered Southern White Crested Hedgehogs
- Peter's Pond, local bathing place in Well's Wood.

This backstory may make the event card descriptions easier to understand and connect.

**Comment by ellenbroad** [comment]
@Stephen-Gates we just made them up. Do you think we need a couple of paragraphs setting the scene? I could add it to the website, but I don't know we'd fit it in the instructions booklet. https://docs.google.com/document/d/1ydC_eXxgZL6VhrBY5mZxI3yfNx58eu6JUJafSb03-rI/edit#

**Comment by Stephen-Gates** [comment]
@ellenbroad When playing with my adult kids (no knowledge of open data), they enjoyed the quirky descriptions but didn't see a strong connection to the game. We started by reading them all out loud but this faded away. I think a back story on the web would help. It may also assist in localisation if people want to create the same dynamic between towns, organisations and characters and not just replace names.

P.S. It got a thumbs up from the kids who are big board game players - congrats on publishing the beta.

**Comment by ellenbroad** [comment]
@Stephen-Gates yay thanks for all of the feedback :). Do you think they're too wordy by the way? This has been an ongoing battle between @JeniT and I - although when she plays it with her kids they like reading through the events.

**Comment by Stephen-Gates** [comment]
@ellenbroad Yep. We did comment that they were too long, too many characters, places, and organisations crammed into some cards - hence the comment on a backstory.
